pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -192,11 +194,16 @@ In particular, we do not share any Personal Information you provide with our don
 
 If your use of the Code.org Website is in the context of a partnership with your school or school district, we may allow the school or school district to access the same student progress report that is also shared with your teacher(s).
 
-### We may share teacher contact information with training facilitators and partner organizations
-Part of our educational program is to offer professional development workshops to prepare teachers to offer computer science courses. If you are a teacher participating in one of these workshops, your name and contact information will be shared with the facilitator and/or professional learning organization hosting the workshop, and they may have access to your continued progress in our online professional learning courses in order to coach or facilitate your learning. If Code.org is paying for your travel to our professional learning workshops for teachers, we will - with your explicit permission - share your name and contact information with our travel partner to facilitate booking your travel.
+### We may share teacher information with training facilitators and our Regional Partners
+
+Part of our educational program is to offer professional development workshops to prepare teachers to offer computer science courses. If you are a teacher participating in one of these workshops, your name and contact information will be shared with the facilitator and/or professional learning organization hosting the workshop, and they may have access to your continued progress in our online professional learning courses in order to coach or facilitate your learning. They may also be able to see your overall class progress to support your classroom. They will not have any data on specific students. If Code.org is paying for your travel to our professional learning workshops for teachers, we will - with your explicit permission - share your name and contact information with our travel partner to facilitate booking your travel.
 
 In the context of partnerships between Code.org and school districts, we may also share this workshop attendance and professional learning data with the school district that employs the teacher, in order to allow school districts to compensate teachers in situations where school districts have such an arrangement with their teachers.
 
+### We may share de-identified information on school or district usage with our Regional Partners
+
+Our Regional local Partners will be able to see de-identified demographic information and usage of courses and professional learning programs for schools and districts in their area. They will not see names or contact information of any teacher unless you choose to share it as part of signing up for a local professional development workshop or joining their professional learning program.
```

Comment:
Is there a reason local was put in the middle? I think we should keep the official name together like `Our local Regional Partners...`

Reply:
Consistency between professional development and professional learning?

Reply:
Whoops, I meant to take the local out. Thanks! And Alice wrote it up like this and I think I remember her telling me it was intentional...so I might leave it as "professional development" for now.